icicle/tests/test_hash_api.cpp

```diff
@@ -1067,3 +1067,51 @@ TEST_F(HashApiTest, poseidon_tree)
 }
 
 #endif // POSEIDON
+
+/*======================= (TODO) Move to Sumcheck test-suite =======================*/
+#include "icicle/sumcheck/sumcheck_transcript_config.h"
+
+class SumcheckTest : public IcicleTestBase
+{
+public:
+  // Helper function to convert a byte vector to a string for comparison
+  static std::string bytes_to_string(const std::vector<std::byte>& bytes)
+  {
+    return std::string(reinterpret_cast<const char*>(bytes.data()), bytes.size());
+  }
+};
+
+TEST_F(SumcheckTest, InitializeWithConstChar)
```

Comment:
test name should be SumcheckTranscriptConfigInitializeWithConstChar
for filtering

Reply:
your name is too short but ok I will rename. Although if I move it to a sumcheck test suite then I can remove the Sumcehck prefix from the test name I guess.

Reply:
do you plan to have a different test suite for FRI?
If so then I agree

Reply:
I think so yes